lms/static/js/spec/bookmarks/bookmarks_list_view_spec.js

```diff
@@ -110,7 +137,7 @@ define(['backbone', 'jquery', 'underscore', 'logger', 'js/common_helpers/ajax_he
 
             it("has rendered empty bookmarks list correctly", function () {
```

Comment:
Nit: change this to match the naming style of our jasmine tests. For example: `it("can correctly render an empty bookmarks list", function () {...});`